app/src/main/java/com/github/kulebin/myfoursquareapp/adapter/VenueListAdapter.java

```diff
@@ -64,7 +70,11 @@ public void displayRating(final String rating) {
 
         @Override
         public void displayImage(final String imageUrl) {
-            //todo load and show image
+            imageView.setImageResource(R.drawable.placeholder_foursquare);
+
+            if (imageUrl != null) {
+                IImageLoader.Impl.get().draw(imageUrl, imageView, IMAGE_WIDTH, IMAGE_HEIGHT);
```

Comment:
Future improvements: 
- use scaled dimensions, or get image size from imageView
- let placeHolder image as a setting or argument, and use in ImageLoader, not when call loader